pkgs/applications/video/byzanz/default.nix

```diff
@@ -5,7 +5,7 @@ stdenv.mkDerivation {
   pname = "byzanz";
 
   src = fetchgit {
-    url = "git://github.com/GNOME/byzanz";
+    url = "https://github.com/GNOME/byzanz";
```

Comment:
Does not exist, should go to `https://gitlab.gnome.org/Archive/byzanz`

Reply:
```suggestion
    url = "https://gitlab.gnome.org/Archive/byzanz";
```